src/routes/RecipeDetails.js

```diff
@@ -6,35 +6,34 @@ class RecipeDetails extends React.Component {
     super(props)
 
     this.state = {
-      recipeDetails: {},
+      recipe: {},
     }
   }
 
   async componentDidMount() {
     const response = await fetch(`/api/recipes/${this.props.match.params.id}.json`)
-    const recipeDetails = await response.json()
-    this.setState({ recipeDetails })
+    const recipe = await response.json()
+    this.setState({ recipe })
   }
 
   render() {
-    const { recipeDetails } = this.state
-    const recipeInfo = recipeDetails.recipeDetail
+    const { recipe, recipe: {recipeDetail} } = this.state
 
     return (
-      <>
-        <div className="text-center container">
-          <h2 className="text-secondary text-uppercase mb-4">{recipeDetails.title}</h2>
-          <p>{recipeDetails.description}</p>
+      <div>
```

Comment:
I had a feeling you were gonna do this 😛 
If there is no class, there is no need for a `div` 😂 
```suggestion
      <>
```